tasks/docker/install.yml

```diff
@@ -4,19 +4,22 @@
   file:
     path: "{{ jenkins_home }}"
     mode: 0777
-    recurse: yes
+    owner: "{{ jenkins_config_owner }}"
+    group: "{{ jenkins_config_group }}"
     state: directory
 
 - name: Container is running
-  docker:
+  docker_container:
```

Comment:
I'd prefer not to force ansible 2.1+ as the minimum version, I personally still use 1.9 for example. Unless there is a really good reason to use the new `docker_container` module can you go back to using the old `docker` module?